mmv1/templates/terraform/examples/cloudbuild_trigger_repo.tf.erb

```diff
@@ -0,0 +1,72 @@
+terraform {
+    required_providers {
+        google-beta = {}
+        google = {}
+    }
+}
+
+provider "google-beta" {}
+
+resource "google_secret_manager_secret" "github-token-secret" {
```

Comment:
Within an example, we use the same provider version throughout. Here, that should be `provider = google-beta` (This applies to all HCL files)